compat/mimalloc/stats.c

```diff
@@ -479,12 +480,17 @@ static void mi_stat_process_info(mi_msecs_t* elapsed, mi_msecs_t* utime, mi_msec
   *utime = filetime_msecs(&ut);
   *stime = filetime_msecs(&st);
   PROCESS_MEMORY_COUNTERS info;
-  GetProcessMemoryInfo(GetCurrentProcess(), &info, sizeof(info));
-  *current_rss    = (size_t)info.WorkingSetSize;
-  *peak_rss       = (size_t)info.PeakWorkingSetSize;
-  *current_commit = (size_t)info.PagefileUsage;
-  *peak_commit    = (size_t)info.PeakPagefileUsage;
-  *page_faults    = (size_t)info.PageFaultCount;
+  DECLARE_PROC_ADDR(psapi, BOOL, WINAPI, GetProcessMemoryInfo, HANDLE, PPROCESS_MEMORY_COUNTERS, DWORD);
+  if (INIT_PROC_ADDR(GetProcessMemoryInfo)) {
+    GetProcessMemoryInfo(GetCurrentProcess(), &info, sizeof(info));
```

Comment:
Sure, that would be a fine (optional) thing to do in Trace2. It should still be delay-loaded just in case the call is not hit (I _think_ it is only called when showing stats), so that we can avoid loading the dynamic library unnecessarily.